### ISSUE TITLE
Error when starting Playground - Corrupted zip: missing 4 bytes.

### ISSUE THREAD

**Issue by kodepareek** [hyperledger/composer#4229]
<General Summary>
Playground initializes with a popup - showing Error: Corrupted zip: missing 4 bytes.

## Context
Have been trying to make a custom install using Cello. But manually upgraded the composer-cli and playground to - latest

## Expected Behavior
The playground should load, allow BNA upload and attachment of cards to participants

## Actual Behavior
1) Playground loads without the PeerAdmin card that has been successfully imported into the wallet by the CLI and displays error - Error: Corrupted zip: missing 4 bytes.
2) Cannot upload BNA files - same error as above displays

## Your Environment
* Version used: HLF V 1.1, composer-cli and plaground - docker images tagged latest
* Operating System and version - 16.4

**Comment by mahoney1** [comment]
please raise a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/hyperledger-composer) with full details including steps taken / screeenshots as appropriate.

Also state 

- whether your Playground is a dockerized container on installed 'natively' eg in a virtual machine, 
- docker ps
-  the output of `composer card list` there. It sounds like your PeerAdmin card is corrupt, please specify how you created that also over on Stack Overflow, thanks ! 

Once the business network is installed (which means you imported a PeerAdmin card successfully to do the install) playground would discover what networks / cards are available in the wallet for the current OS user.

(Issues is usually for enhancement requests or bug fix requests and the latter is not established)

**Comment by kodepareek** [comment]
Hello there. I have terminated those AWS instances and gone ahead with a much hackier solution to proceed with my UI development. I will not be raising the question on SO as I will not be able to verify any solutions. 

Also state

- whether your Playground is a dockerized container or installed 'natively' eg in a virtual machine - Both CLI and Composer-Playground are dockerized images tagged latest - on docker hub.
- docker ps output - shows that playground in running as expected.
- the output of composer card list there - was displaying the created card in the wallet. Cannot paste output now unfortunately. 

Closing this issue.
